config/database.yml

```diff
@@ -7,7 +7,7 @@ default: &default
 
 development:
   <<: *default
-  database: goodcity_server_development
+  database: goodcity_server_development_2
```

Comment:
@patrixr Can you please rollback this change?

Reply:
already did